include/internal/list.h

```diff
@@ -41,12 +41,12 @@
     static ossl_unused ossl_inline type *                                   \
     ossl_list_##name##_head(const OSSL_LIST(name) *list)                    \
     {                                                                       \
-        return list->head;                                                  \
+        return list->alpha;                                                 \
```

Comment:
They are designed to be esoteric so they don't match what you are using elsewhere.  Otherwise conversion is more difficult.  They can be changed back once everything has been converted.